diagnostic.js

```diff
@@ -32,7 +34,8 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update({"house": "Stark"}, {$set: {"members": ["Ned Stark", "Arya Stark", "Sansa Stark", "Viserys Targaryen", "Daenerys Targaryen"]} });
```

Comment:
you're using exactly the right syntax for `update`, but you are adding ALL the members to House Stark

```js
db.houses.update(
  {name: "House Stark"},
  {$set: { members: [ "Ned Stark", "Arya Stark", "Sansa Stark"] }}
);
```